classes/shortcodes.php

```diff
@@ -68,10 +68,11 @@ function get_shortcode_tags() {
 				'phone',
 				'fax',
 				'email',
-				'url'
+				'url',
 			);
 
-			return apply_filters( 'sm-location-shortcode-tag-array', $shortcode_tags );
+			return apply_filters( 'sm-location-shortcode-tag-array',
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Opening parenthesis of a multi-line function call must be the last content on the line

<!-- Comment by [SideCI](https://sideci.com) -->